src/manage/commands.py

```diff
@@ -476,73 +514,62 @@ def execute(self):
 
     @classmethod
     def usage_text_lines(cls):
-        usage_docs = [
-            (f"    {EXE_NAME} -V:!B!<TAG>!W!",
-             "Launch runtime identified by !B!<TAG>!W!, which should include the " +
-             "company name if not !B!PythonCore!W!. Regular Python options may " +
-             "follow this option."),
-            (f"    {EXE_NAME} -!B!<VERSION>!W!",
-             r"Equivalent to -V:PythonCore\!B!<VERSION>!W!. The version must " +
-             "begin with the digit 3, platform overrides are permitted, " +
-             "and regular Python options may follow." +
-             (" !G!py -3!W! is the equivalent of the !G!python3!W! command." if EXE_NAME == "py" else "")),
-            (f"    {EXE_NAME} !B!<COMMAND>!W!",
-             "Run a specific command (see list below)."),
-        ]
-
-        usage_ljust = max(len(logging.strip_colour(i[0])) for i in usage_docs)
+        if EXE_NAME.casefold() in ("py".casefold(), "pyw".casefold()):
```

Comment:
`py` and `pyw` are already casefolded?
```suggestion
        if EXE_NAME.casefold() in {"py", "pyw"}:
```

Reply:
I try not to assume that lowercase literals are casefolded. I'd much prefer a case-insensitive comparison anyway.